.idea/inspectionProfiles/Gradle.xml

```diff
@@ -20,6 +20,7 @@
     <inspection_tool class="LambdaCanBeMethodCall" enabled="false" level="INFORMATION" enabled_by_default="false" />
     <inspection_tool class="SafeVarargsDetector" enabled="false" level="WARNING" enabled_by_default="false" />
     <inspection_tool class="SimplifyForEach" enabled="false" level="INFORMATION" enabled_by_default="false" />
+    <inspection_tool class="Since15" enabled="true" level="WEAK WARNING" enabled_by_default="true" />
```

Comment:
Why change this?

Reply:
I accidentally committed it. I will remove the change.